yearn/treasury/treasury.py

```diff
@@ -62,21 +62,24 @@ def _get_price(token, block=None):
 
 def get_token_from_event(event):
     try:
-        address = event['Transfer'][0].address
+        if '(unknown)' in event:
```

Comment:
this part is unneeded, the unknown error comes from wrong LINK abi cached, need to find the root cause of it

Reply:
removed it, once it pops up again we know what to do.